### DISCUSSION TITLE
Can't add my domain

### DISCUSSION THREAD

**Discussion by cameololm** [ant-media/Ant-Media-Server#5933] [Q&A]
Hi,

Previously, I created a VPS and installed AMS and successfully added my domain (I used [#Nginx reverse proxy](https://github.com/orgs/ant-media/discussions/4983#discussioncomment-5919916)). After a period of use, I deleted that VPS and created a new one.

After creating a new VPS, I installed AMS and added my old domain to AMS, it still displayed the success message but it did not automatically refresh the page.

I tried reloading the page, it wasn't like the first time I installed AMS, it didn't take me to the login page but it was still on the settings page, I clicked on the SSL section on the page and didn't see any domains added.

I tried accessing using the old domain but it didn't work.

Please help me

**Comment by USAMAWIZARD** [comment]
Hi @cameololm,
Can you please check if the Ngnix was configure properly , 

nginx -t

Can you please share the nginx config file

**Reply by cameololm** [reply]
Hi,
What I mean is, after successfully installing AMS, I can't add my domain name in SSL settings on the admin page
That domain I used before on the old AMS that I deleted, that old AMS I used Nginx reverse proxy
Now I recreated a new AMS and can't add that old domain

**Reply by USAMAWIZARD** [reply]
If you are comfortable can you please share me your domain and Dashboard Credentials at usama.tahseen@antmedia.io

**Reply by cameololm** [reply]
I don't understand why but today that problem was resolved
Maybe it's because I changed my username/Email when registering AMS account
